dune

```diff
@@ -1,10 +1,12 @@
 ; Default flags for all Coq libraries.
 (env
  (dev     (flags :standard -rectypes -w -9-27-50+40+60))
- (release (flags :standard -rectypes))
+ (release (flags :standard -rectypes)
+          (ocamlopt_flags -O3 -unbox-closures))
  (ireport
   (flags :standard -rectypes -w -9-27-50+40+60)
-  (ocamlopt_flags :standard -O3 -unbox-closures -inlining-report)))
+  (ocamlopt_flags :standard -O3 -unbox-closures -inlining-report))
+ (_       (flags :standard -rectypes)))
```

Comment:
I still think that this line is useless if we have to repeat the same thing everywhere. You even repeat it in `(release)` which has exactly the same `flags`.
And what is this very weird alignment after the merge?

Reply:
> And what is this very weird alignment after the merge?

The alignment is fine AFAICS.

Regarding using `(_ ...)` I dunno, I am not sure what it does, my understanding indeed is that it would help for profiles that are local to some part of the sub-tree.

Reply:
>Regarding using (_ ...) I dunno, I am not sure what it does, my understanding indeed is that it would help for profiles that are local to some part of the sub-tree.

If you are not sure, just don't add it for now. We have enough of code that we don't understand fully.